deploy/README_HELM.md

```diff
@@ -0,0 +1,343 @@
+# Deploying TopoLVM
+
+> :memo:
+>
+> This document is an adaptation of the existing README.md to Helm Chart.
+> After removing the kustomize manifests, this document will be renamed to README.md.
+
+Each of these steps are shown in depth in the following sections:
+
+1. Deploy [lvmd][] as a `systemd` service on a worker node with LVM installed.
+1. Prepare [cert-manager][] for [topolvm-controller][]. You may supplement an existing instance.
+1. Determine how [topolvm-scheduler][] to be run:
```

Comment:
Now, we have 3 options, use capacity tracking, or use topolvm-scheduler as a deployment or  use topolvm-scheduler as a daemonset.

Reply:
Thank you comments.
I edited the `deploy.md` to reflect your comments.
Please re-review 🙏 
https://github.com/topolvm/topolvm/pull/325/commits/88ed3cd7cb63222c36905fca71196255fc10e052